www/templates/about.html

```diff
@@ -19,4 +19,6 @@ <h2>{{ lang.about.stars }}</h2>
 <p>{{ lang.about.stars_fame }}</p>
 <h2>{{ lang.about.leaderboard }}</h2>
 <p>{{ lang.about.leaderboard_content }}</p>
+<h2>{{ lang.about.contribute }}</h2>
+<p>{{ lang.about.contribute_content }}</p>
```

Comment:
You should add `| safe` after the variable, so html entities are preserved.

Reply:
Done!